R/removeHyperPars.R

```diff
@@ -23,6 +23,15 @@ removeHyperPars = function(learner, ids = character(0L)) {
 #' @export
 removeHyperPars.Learner = function(learner, ids = character(0L)) {
   learner$par.vals[ids] = NULL
+  on.par.out.of.bounds = coalesce(learner$config$on.par.out.of.bounds, getMlrOptions()$on.par.out.of.bounds)
```

Comment:
this whole code block seems strage. discuss this with BB and mllg in review. but please:
never do this! please ALWAYS add comments here. i am not even sure whether you should have also extended the docs.
problem is: we dont get at all what happens here.